WordPress/Classes/ViewRelated/Blog/Site Monitoring/WebServerLogsView.swift

```diff
@@ -214,7 +214,7 @@ final class WebServerLogsViewModel: ObservableObject {
 
         do {
             let endDate = searchCriteria.endDate ?? Date.now
-            let startDate = searchCriteria.startDate ?? (Calendar.current.date(byAdding: .weekOfYear, value: -1, to: endDate) ?? endDate)
+            let startDate = searchCriteria.startDate ?? Date.oneWeekAgo(from: endDate)
```

Comment:
Thanks for cleaning this up. I initially wanted to remove `oneWeekAgo`, but noticed it was also used as a defaultt value, so I had to keep it.